packages/earth-admin/src/layouts/Sidebar.tsx

```diff
@@ -17,40 +17,61 @@
   specific language governing permissions and limitations under the License.
 */
 
-import React from 'react';
+import React, { useContext } from 'react';
+import { navigate } from 'gatsby';
 
-import { Spinner } from '@marapp/earth-shared';
+import { Spinner, AppContextSwitcher } from '@marapp/earth-shared';
 
 import { LinkWithOrg } from '@app/components/link-with-org';
-import { OrgSwitcher } from '@app/components/org-switcher';
 import { SidebarSelect } from '@app/components/sidebar-select';
 import { IAdminPage } from '@app/components/sidebar-select/model';
+import { Auth0Context } from '@app/utils/contexts';
 
 import { APP_LOGO, APP_NAME } from '../theme';
 import './styles.scss';
 
+const { Option } = AppContextSwitcher;
+
 interface IProps {
   isLoading?: boolean;
   children?: React.ReactNode;
   page?: IAdminPage[];
 }
 
 const SidebarLayout = (props: IProps) => {
+  const { groups, selectedGroup } = useContext(Auth0Context);
+
+  const logo = (
+    <LinkWithOrg to="/" className="ng-border-remove">
+      <img src={APP_LOGO} alt={APP_NAME} className="ng-margin-remove ng-display-block" />
+    </LinkWithOrg>
+  );
+
   return (
     <div className="ng-sidebar ng-flex ng-flex-column ng-flex-top ng-shadow-medium">
-      <div className="ng-shadow-large ng-background-dkgray">
-        <nav className="ng-padding-medium-top ng-sidebar-header">
-          <div className="ng-padding-medium-horizontal ng-ep-background-dark ng-margin-bottom ng-flex ng-flex-middle ng-position-relative">
-            <LinkWithOrg to="/" className="ng-border-remove">
-              <img src={APP_LOGO} alt={APP_NAME} className="ng-margin-remove ng-display-block" />
-            </LinkWithOrg>
-            <span className="ng-margin-small-horizontal ng-color-white">|</span>
-            <OrgSwitcher />
-          </div>
-        </nav>
+      <div className="ng-sidebar-header ng-shadow-large ng-background-dkgray ng-padding-bottom">
+        <AppContextSwitcher
+          logo={logo}
+          label="Map View"
+          defaultValue={selectedGroup}
+          onChange={(g) => {
+            if (g === 'map-view') {
+              window.location.assign('/');
```

Comment:
pls use env variable GATSBY_APP_MAP_URL
should this link to /earth?

Reply:
good catch